post_office/mail.py

```diff
@@ -40,6 +42,10 @@ def create(sender, recipients=None, cc=None, bcc=None, subject='', message='',
         bcc = []
     if context is None:
         context = ''
+    if get_message_id_on_delivery():
+        message_id = None
+    else:
+        message_id = make_msgid(domain=get_message_id_fqdn())
```

Comment:
We can store the message_id without the domain part, so this can save a bit of space. Since it's a UUID, we can already uniquely identify an email without the domain part.

The FQDN part can be added when the mail is being delivered.